services/build-images.sh

```diff
@@ -45,18 +50,18 @@ print_used_proxies() {
 build_docker_image() {
   module_name_base=$(echo "$1" | awk -F ":" '{ print $1 }')
   module_name=$(printf "ditto-services-%s" "$module_name_base")
-  image_tag=$(printf "eclipse/ditto-%s" "$(echo "$1" | awk -F ":" '{ print $2 }')")
+  image_tag=$(printf "${CONTAINER_REGISTRY}/ditto-%s" "$(echo "$1" | awk -F ":" '{ print $2 }')")
   printf "\nBuilding Docker image <%s> for service module <%s>\n" \
     "$image_tag" \
     "$module_name"
 
-    $DOCKER_BIN build --pull -f $DOCKERFILE \
+    $DOCKER_BIN build --pull -f $SCRIPTDIR/$DOCKERFILE \
       --build-arg HTTP_PROXY="$HTTP_PROXY_LOCAL" \
       --build-arg HTTPS_PROXY="$HTTPS_PROXY_LOCAL" \
       --build-arg TARGET_DIR="$module_name_base"/starter/target \
       --build-arg SERVICE_STARTER="$module_name"-starter \
       --build-arg SERVICE_VERSION=$SERVICE_VERSION \
-      -t "$image_tag":$SERVICE_VERSION \
+      -t "$image_tag":$IMAGE_VERSION \
       $DOCKER_BUILD_CONTEXT
```

Comment:
You removed the `DOCKER_BUILD_CONTEXT` variable - can this reference `$DOCKER_BUILD_CONTEXT` then also be removed?

Reply:
Indeed. I replaced it with `$SCRIPTDIR` now.